src/environ.c

```diff
@@ -208,7 +208,7 @@ static int setup_cfg_vars(struct sandbox_info_t *sandbox_info)
 	if (-1 == setup_access_var(ENV_SANDBOX_WRITE))
 		return -1;
 	if ((NULL == getenv(ENV_SANDBOX_WRITE)) &&
-	    (NULL != sandbox_info->work_dir))
+	    strlen(sandbox_info->work_dir))
```

Comment:
how do you know `sandbox_info->work_dir` isn't `NULL`?

Reply:
Because src/sandbox.h:22 defines work_dir as char[SB_PATH_MAX]  as a member of struct sandbox_info_t.

Reply:
yup, didn't check the definitions, :+1: